public/app/types/explore.ts

```diff
@@ -125,12 +119,6 @@ export interface ExploreItemState {
    */
   scanRange?: RawTimeRange;
 
-  loading: boolean;
```

Comment:
Removed the loading to simplify the ExplorePanelProps, seems like it is easily derived from loadginState